scalableminds/webknossos
# Fix creating a task using a base NML

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open the "create a task" dialog
- select "Upload NML File" and upload an NML
- fill the remaining values
- create a task --> should work

### Issues:
- follow-up for https://github.com/scalableminds/webknossos/pull/6486/files

------
(Please delete unneeded items, merge only when none are left open)
